scripts/maintenance/migrate_project/src/models.py

```diff
@@ -15,10 +14,17 @@ class DBConfig(BaseModel):
 
 
 class S3Config(BaseModel):
-    endpoint: Optional[str] = None
+    endpoint: str = "https://s3.amazonaws.com"
+    provider: str = Field(
+        ...,
+        description='The S3 implementation / provider. Allowed values: "MINIO","CEPH","AWS"',
```

Comment:
suggestion: using pydantic you can define an enum here
suggestion2: also for the endpoint you can use a ```AnyUrl``` or so type

Reply:
thx but postboned indefinately for now since this is a hacky script anyway. Proper copying should go through storage and eventually db migration.